src/http/ldp/PostOperationHandler.ts

```diff
@@ -38,9 +39,10 @@ export class PostOperationHandler extends OperationHandler {
       throw new BadRequestHttpError('POST requests require the Content-Type header to be set');
     }
     const result = await this.store.addResource(operation.target, operation.body, operation.conditions);
-    const createdIdentifier = Object.entries(result).find(
-      ([ , value ]): boolean => value.get(SOLID_AS.terms.Activity)?.value === AS.Create,
-    )![0];
-    return new CreatedResponseDescription(createResourceIdentifier(createdIdentifier));
+    const createdEntry = find(result, ([ , value ]): boolean => value.has(SOLID_AS.terms.Activity, AS.terms.Create));
```

Comment:
Can we have more descriptive names such that `result, ([ , value ])` can be interpreted on sight?

I think `result` can be `changes`.